Apps/WebClient/src/ClientApp/src/components/DatePickerComponent.vue

```diff
@@ -126,6 +131,7 @@ export default class DatePickerComponent extends Vue {
             <b-form-datepicker
                 ref="datePicker"
                 v-model="value"
+                :max="maxDate"
```

Comment:
I see you've added maxDate to :max in the b-form-datepicker.  

How does this interact with the current validation routine already in place from lines 94 to 113. Wouldn't the prop added on line 24 be used to override the max value defined on line 105?

<img width="975" alt="Screenshot 2023-03-06 at 5 08 50 PM" src="https://user-images.githubusercontent.com/58790456/223292964-bd8608b2-0221-4644-8eb7-a947840e13fa.png">

Does this change have any affect on the date picker used in the timeline filter, report filter and add notes?

Reply:
That's a good point, maybe it should be using the value of maxDate on line 29 as the validator maxValue it is checking against. The maxDate is set to a default value that is the same as the maxValue in the validator, then in the mounted() function if there is a maxPickerDate passed in it sets maxDate to match.

There is no affect on other places that date picker is used since the maxPickerDate is nullable and therefore optional value and is only used if it has been set. So I didn't set the maxPickerDate anywhere else but in Add Dependent so it will continue to work the same as before.

Reply:
Fixed so that the maxDate is used for the validation.